pypiserver/_app.py

```diff
@@ -362,6 +363,25 @@ def server_static(filename):
 
     return HTTPError(404, f"Not Found ({filename} does not exist)\n\n")
 
+@app.route("/:project/json")
+@auth("list")
+def json_info(project):
```

Comment:
[nit:] It would be grand (instant karma points) if you could add a unit test for this at `tests/test_app.py`.